code/__HELPERS/_logging.dm

```diff
@@ -151,6 +151,10 @@
 	if (CONFIG_GET(flag/log_telecomms))
 		WRITE_LOG(GLOB.world_telecomms_log, "TCOMMS: [text]")
 
+/proc/log_network(text)
+	if (CONFIG_GET(flag/log_network))
+		WRITE_LOG(GLOB.network_log, "NETWORK: [text]")
```

Comment:
```suggestion
#DEFINE LOG_NETWORK(text) if (CONFIG_GET(flag/log_network)) { WRITE_LOG(GLOB.network_log, "NETWORK: [text]") }
		
```
Optional change. I know it's not how it's done for the other procs, but there's no reason for these to be procs instead of defines.